cmd/scaffold.go

```diff
@@ -682,23 +695,32 @@ func FlowNode(role string) *FlowNodeBuilder {
 
 	builder := &FlowNodeBuilder{
 		BaseConfig: BaseConfig{
-			nodeRole: role,
+			NodeRole: role,
 		},
 		Logger: zerolog.New(os.Stderr),
 		flags:  pflag.CommandLine,
 	}
 
-	builder.baseFlags()
+	return builder
+}
 
-	builder.enqueueNetworkInit()
+func (fnb *FlowNodeBuilder) Initialize() *FlowNodeBuilder {
```

Comment:
broken the constructor `FlowNode` into `FlowNode` and `Initialize`. For the access node, the initialization is different and breaking down the function helps that custom initialization.